pkg/server/api_plot.go

```diff
@@ -357,7 +357,9 @@ func makePlotsResponse(plotSeries map[string][]plot.Series, plotReq *PlotRequest
 					plotItem.Scale(plot.Value(scale))
 				}
 
-				seriesOptions[seriesItem.Name] = seriesItem.Options
+				if len(seriesItem.Options) != 0 {
```

Comment:
Hi Matthias,

Ok it makes sense and I understand your concern now.

Indeed, there is a mix-up in options handling and we are trying to fix that and make it consistent. But we still have some work to do.

There is some use cases where options are needed at series level and some at group level and even sometimes both, by example:
 * different `unit` on multiple series then grouped into a _NORMALIZE_ group
 * `unit` on group of type _SUM_ or _AVG_
 * `scale` on series and then another one on group

I guess we could by default inherit options from group (merge them) giving the priority to the ones defined in series.

But we still have to figure a way to allow series options edition in the UI once they are grouped. Which is currently impossible :(

We are open to suggestions.

 Thanks for the explanation :)